src/utils/telemetry.js

```diff
@@ -46,23 +46,18 @@
 
 import { Telemetry } from "devtools-modules";
 
-const telemetry = new Telemetry();
+let telemetry = null;
+
+export function setupTelemetry(id: number) {
+  telemetry = new Telemetry(id);
```

Comment:
thanks, my guess is that `Telemetry` does not currently accept `id` in the constructor and we'll have to add that. Is that correct @MikeRatcliffe?